### PR TITLE
migrating ProxySQL to community.proxysql

### PR DESCRIPTION
**Pull request by bmildren**
##### SUMMARY
Ready to merge once community.proxysql has been released.

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
proxysql_*

### CODE REVIEW

File: changelogs/fragments/proxysql.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+removed_features:
+- "proxysql_* - all ProxySQL modules have been moved to the ``community.proxysql`` collection. A redirection is active, which will be removed in version 2.0.0. (https://github.com/ansible-collections/community.general/pull/624)"
```
Comment (felixfontein) [inline changelogs/fragments/proxysql.yml:2]:
```suggestion
- "proxysql_* - all ProxySQL modules have been moved to the ``community.proxysql`` collection. A redirection is active, which will be removed in version 2.0.0 (https://github.com/ansible-collections/community.general/pull/624)." 
```

**Comment by felixfontein** [comment]
You also need to remove the now dead symlinks:
```

plugins/modules/proxysql_backend_servers.py:0:0: broken symlinks are not allowed
plugins/modules/proxysql_global_variables.py:0:0: broken symlinks are not allowed
plugins/modules/proxysql_manage_config.py:0:0: broken symlinks are not allowed
plugins/modules/proxysql_mysql_users.py:0:0: broken symlinks are not allowed
plugins/modules/proxysql_query_rules.py:0:0: broken symlinks are not allowed
plugins/modules/proxysql_replication_hostgroups.py:0:0: broken symlinks are not allowed
plugins/modules/proxysql_scheduler.py:0:0: broken symlinks are not allowed
```

File: changelogs/fragments/proxysql.yml (hunk 1 above)
Comment (bmildren) [reply changelogs/fragments/proxysql.yml:2]: 👍

**Comment by bmildren** [comment]
Thanks for the feedback Felix!  As suggested, I've removed the symlinks and fixed the grammar 👍

**Comment by gundalow** [comment]
https://galaxy.ansible.com/community/proxysql has been published, so merging.
@bmildren @felixfontein Amazing work, thank you!

**Pull request merged**